pbjson/src/lib.rs

```diff
@@ -38,4 +42,41 @@ pub mod private {
             }))
         }
     }
+
+    #[derive(Debug, Copy, Clone, PartialOrd, PartialEq, Hash, Ord, Eq)]
+    pub struct BytesDeserialize<T>(pub T);
+
+    impl<'de, T> Deserialize<'de> for BytesDeserialize<T>
+    where
+        T: From<Vec<u8>>,
+    {
+        fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+        where
+            D: serde::Deserializer<'de>,
+        {
+            let str: &str = Deserialize::deserialize(deserializer)?;
+            let decoded = base64::decode(str).map_err(serde::de::Error::custom)?;
```

Comment:
```suggestion
            let s: &str = Deserialize::deserialize(deserializer)?;
            let decoded = base64::decode(s).map_err(serde::de::Error::custom)?;
```

Could we please not shadow builtins or use types as variable names?